{
  "eventVersion": "1.08",
  "userIdentity": {
    "type": "AssumedRole",
    "principalId": "AROA2KEX3JLIQNOCCAZIP:ebs_eni_switch_AAC",
    "arn": "arn:aws:sts::999999999999:assumed-role/NML-Lambda-ebs_eni_switch/ebs_eni_switch_AAC",
    "accountId": "999999999999",
    "accessKeyId": "ASIA2KEX3JLI6VQRWL55",
    "sessionContext": {
      "sessionIssuer": {
        "type": "Role",
        "principalId": "AROA2KEX3JLIQNOCCAZIP",
        "arn": "arn:aws:iam::999999999999:role/NML-Lambda-ebs_eni_switch",
        "accountId": "999999999999",
        "userName": "NML-Lambda-ebs_eni_switch"
      },
      "webIdFederationData": {},
      "attributes": { "mfaAuthenticated": "false", "creationDate": "2021-04-27T12:07:04Z" }
    }
  },
  "eventTime": "2021-04-27T12:07:06Z",
  "eventSource": "ec2.amazonaws.com",
  "eventName": "AttachVolume",
  "awsRegion": "ap-northeast-1",
  "sourceIPAddress": "18.181.177.86",
  "userAgent": "Boto3/1.17.42 Python/3.7.10 Linux/4.14.225-175.364.amzn2.x86_64 exec-env/AWS_Lambda_python3.7 Botocore/1.20.42",
  "errorCode": "Client.InvalidVolume.ZoneMismatch",
  "errorMessage": "The volume 'vol-02d2ff61f0b482457' is not in the same availability zone as instance 'i-0bab343c995dda5a4'",
  "requestParameters": {
    "volumeId": "vol-02d2ff61f0b482457",
    "instanceId": "i-0bab343c995dda5a4",
    "device": "/dev/sdi",
    "deleteOnTermination": false
  },
  "responseElements": null,
  "requestID": "0c056615-da26-4454-93b1-8aff1a308aba",
  "eventID": "435fb93c-9bc4-476e-a286-178629d0b90e",
  "readOnly": false,
  "eventType": "AwsApiCall",
  "managementEvent": true,
  "eventCategory": "Management",
  "recipientAccountId": "999999999999"
}
